parser.mly

```diff
@@ -25,15 +25,15 @@ let scope_annotation (mode, xs) =
 %%
 
 program:
-| prog=list(instruction_line) EOF
+| list(NEWLINE) prog=list(instruction_line) EOF
   {
     let annotations, instructions = List.split prog in
     (Array.of_list instructions,
      Array.of_list annotations)
   }
 
 instruction_line:
-| a=scope_annotation i=instruction NEWLINE { (a, i) }
+| a=scope_annotation i=instruction nonempty_list(NEWLINE) { (a, i) }
```

Comment:
would it be unambiguous (make sense) to also allow newlines between scopes and instruction?